### PR TITLE
[ADD] Tensorboard

### PR DESCRIPTION
**Pull request by LEEYEONSU**
tensorboard
- train/acc
- train/loss
- val/acc
- val/loss
- lr
요렇게만 일단 추가했옹

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -8,14 +8,16 @@
...
+from tensorboardX import SummaryWriter
 
 import theconf
 from theconf import Config as C
 
 import trainer
 
+summary = SummaryWriter()
 def main(flags):
+
```
Comment (justHungryMan) [inline train.py:20]: 지우자

File: train.py [hunk 2]
```diff
@@ -60,13 +62,21 @@ def main(flags):
...
 
         print(f'THE BEST MODEL val test :{is_best:3f}')
+                
+    ### tensorboard
+        summary.add_scalar("Loss/train", train_loss, epoch)
+        summary.add_scalar("Acc/train", train_acc, epoch)
+        summary.add_scalar("Loss/val", val_loss, epoch)
+        summary.add_scalar("Acc/val", val_acc, epoch)
+        summary.add_scalar("learning_rate", optimizer.param_groups[0]["lr"], epoch)
+        summary.close()
```
Comment (justHungryMan) [inline train.py:79]: 여기 summary.close() 맞어?

Comment (LEEYEONSU) [reply train.py:79]: close를 안하면 저장이 안된다고 해서 꼭 써야되는걸로 알아서 추가했엉
텐서보드 작동  잘 됩니당

File: train.py (hunk 1 above)
Comment (LEEYEONSU) [reply train.py:20]: 난 한칸 띄는게 더 깔끔해보이던데 ... 알겠어

File: train.py (hunk 2 above)
Comment (LEEYEONSU) [reply train.py:79]: 실제로 close 안하거나 for 문 밖에서 close 해봤는데 텐서보드에 업데이트가 안되서 그래프가 안그려지더라구

**Review by justHungryMan — APPROVED** [review]
